pkg/registry/rbac/validation/rule.go

```diff
@@ -212,6 +216,14 @@ func (r *DefaultRuleResolver) VisitRulesFor(user user.Info, namespace string, vi
 		} else {
 			sourceDescriber := &roleBindingDescriber{}
 			for _, roleBinding := range roleBindings {
+
+				// verify user tenant matches rolebinding tenant
+				// "to remove check for default later when rolebinding can 
```

Comment:
Need to make changes to to get the rolebindings only for the given userTenant. We may have tens of thousands of userTenants, it is very inefficient if we check all the role bindings for all the tenants.

Reply:
your changes have been picked up for both rolebinding and clusterrolebinding. thanks!